semantic-reflex/src/Reflex/Dom/SemanticUI/Dropdown.hs

```diff
@@ -326,11 +327,18 @@ searchDropdown config@DropdownConfig {..} ini items = mdo
 
   -- The user chooses an option by clicking it or "hovering" with the
   -- keyboard and pressing enter
-  let choose :: Event t Int = fmapMaybe id $ leftmost
+  let chooseIdx :: Event t Int = fmapMaybe id $ leftmost
         [ clickIndex
         , tag (current hover) $ keydown Enter dropdownElement
         ]
 
+      choose :: Event t (k, Text)
+      choose = fmapMaybe id $ attachWith (&)
+        (taggedActive constant current itemMap)
```

Comment:
Will doing it this way make the `TaggedStatic` case slower than with `ffor`?

Reply:
I'd guess so, but it's probably negligible